src/main/java/pl/javastart/task/Shop.java

```diff
@@ -0,0 +1,32 @@
+package pl.javastart.task;
+
+import java.util.Arrays;
+
+public class Shop {
+    public static void main(String[] args) {
+        ShoeBuilder builder = new ShoeBuilder();
+        Shoe[] shoes = new Shoe[3];
+        for (int i = 0; i < shoes.length; i++) {
+            shoes[i] = builder.createShoe();
+        }
+        for (int i = 0; i < shoes.length; i++) {
+            for (int j = 1; j < shoes.length; j++) {
+                if (shoes[i].equals(shoes[j])) {
+                    System.out.println("Duplikat");
+                    shoes[j] = builder.createShoe();
+                }
+            }
+        }
```

Comment:
Z tego, co rozumiem, to jeśli 2x stworzę duplikat w tym samym miejscu, to tablica nadal będzie je zawierała.
Trzeba to zrobić tak, że będziemy pytać użytkownika do oporu, aż poda wartkość, która nie jest duplikatem.